src/BlockchainData.js

```diff
@@ -0,0 +1,228 @@
+		multisigERC20Token.methods.tokenBalances(this.state.etherAddress, TOKENIMPERIAL_SYMBOL).call({from: this.state.etherAddress}, (error, balance) => {
+			if (error) {
+		        console.log(error)
+		    } else {
+		        this.setState({ tokenImperialContractBalance: balance });
+		       	console.log('The imperial token contract balance of account is: ', this.state.tokenImperialContractBalance);
+		    }
+		})
+
+		multisigERC20Token.methods.tokenBalances(this.state.etherAddress, TOKENDEMOCRATIC_SYMBOL).call({from: this.state.etherAddress}, (error, balance) => {
+			if (error) {
+		        console.log(error)
+		    } else {
+		        this.setState({ tokenDemocraticContractBalance: balance });
+		       	console.log('The democratic token contract balance of account is: ', this.state.tokenDemocraticContractBalance);
+		    }
+		})
+
+		multisig.methods.balances(this.state.etherAddress).call({from: this.state.etherAddress}, (error, balance) => {
+		    if (error) {
+		        console.log(error)
+		    } else {
+		        this.setState({ contractBalance: balance });
+		       	console.log('The contract balance of account is: ', this.state.contractBalance);
+		    }
+		})
+
+		tokenImperial.methods.balanceOf(this.state.etherAddress).call({from: this.state.etherAddress}, (error, balance) => {
+		    if (error) {
+		        console.log(error)
+		    } else {
+		        this.setState({ tokenImperialBalance: balance });
+		       	console.log('The token Imperial balance of account is: ', this.state.tokenImperialBalance);
+		    }
+		})
+
+		tokenDemocratic.methods.balanceOf(this.state.etherAddress).call({from: this.state.etherAddress}, (error, balance) => {
+		    if (error) {
+		        console.log(error)
+		    } else {
+		        this.setState({ tokenDemocraticBalance: balance });
+		       	console.log('The token Democratic balance of account is: ', this.state.tokenDemocraticBalance);
+		    }
+		})
+
+		//this.interval = setInterval(this.updateBalances, 15000);
+		//this.interval = updateERCBalances(this.updateERCBalances, 15000);
+  	}
+
+  	componentWillUnmount() {
+	   	clearInterval(this.interval);
+	}
+
+	updateBalances = () => {
+  		let etherBalance = null;
+  		let contractBalance = null
+	    this.state.multisig.methods.balances(this.state.etherAddress).call({ from: this.state.etherAddress }, (error, balance) => {
+        	if (error) {
+	    		console.log(error);
+	    	} else {
+	    		contractBalance = balance;
+	    		this.state.web3.eth.getBalance(this.state.etherAddress, (error, balance) => {
+		  			if (error) {
+		  				console.log(error, 'Something went wrong');
+		  			} else {
+						etherBalance = balance;
+						console.log(etherBalance);
+						console.log('Ether',etherBalance);
+			      		this.setState({ contractBalance : contractBalance, etherBalance : etherBalance});
+			      		console.log('New etherBalance : ', this.state.etherBalance);
+			      		console.log('New contractBalance : ', this.state.contractBalance);
```

Comment:
you can do `const { . etherBalance,  contractBalance, } = this.state;`
to avoid extra code.
Same you can do for any object, props, and other state variables.